docs/dev/python/user-guide/table/10_minutes_to_table_api.md

```diff
@@ -0,0 +1,719 @@
+
+from pyflink.table import DataTypes
+table = table_env.from_elements([(1, 'Hi'), (2, 'Hello')],
+                                DataTypes.ROW([DataTypes.FIELD("id", DataTypes.TINYINT()),
+                                               DataTypes.FIELD("data", DataTypes.STRING())]))
+# now the type of the "id" column is 8 bit int
+type = table.to_pandas()["id"].dtype
+print('Now the type of the "id" column is %s.' % type)
+
+{% endhighlight %}
+
+The result is:
+
+{% highlight text %}
+By default the type of the "id" column is int64.
+Now the type of the "id" column is int8.
+{% endhighlight %}
+
+### Create using a Connector
+
+You can create a Table using connector DDL:
+
+{% highlight python %}
+
+table_env.execute_sql("""
+    CREATE TABLE random_source (
+        id BIGINT, 
+        data TINYINT 
+    ) WITH (
+        'connector' = 'datagen',
+        'fields.id.kind'='sequence',
+        'fields.id.start'='1',
+        'fields.id.end'='3',
+        'fields.data.kind'='sequence',
+        'fields.data.start'='4',
+        'fields.data.end'='6'
+    )
+""")
+table = table_env.from_path("random_source")
+table.to_pandas()
+
+{% endhighlight %}
+
+The result is:
+
+{% highlight text %}
+   id  data
+0   2     5
+1   1     4
+2   3     6
+{% endhighlight %}
+
+### Create using a Catalog
+
+A TableEnvironment maintains a map of catalogs of tables which are created with an identifier.
+
+The tables in a catalog may either be temporary, and tied to the lifecycle of a single Flink session, or permanent, and visible across multiple Flink sessions and clusters.
+
+The tables and views created via SQL DDL, e.g. "create table ..." and "create view ..." are also stored in a catalog.
+
+You can directly access the tables in a catalog via SQL.
+
+If you want to use tables from a catalog with the Table API, you can use the "from_path" method to create the Table API objects:
+
+{% highlight python %}
+
+# prepare the catalog
+# register Table API tables in the catalog
+table = table_env.from_elements([(1, 'Hi'), (2, 'Hello')], ['id', 'data'])
+table_env.create_temporary_view('source_table', table)
+
+# create Table API table from catalog
+new_table = table_env.from_path('source_table')
+new_table.to_pandas()
+
+{% endhighlight %}
+
+The result is:
+
+{% highlight text %}
+   id   data
+0   1     Hi
+1   2  Hello
+{% endhighlight %}
+
+{% top %}
+
+Write Queries
+-------------
+
+### Writing Table API Queries
+
+The `Table` object offers many methods for applying relational operations. 
+These methods return new `Table` objects representing the result of applying the relational operations on the input `Table`. 
+These relational operations may be composed of multiple method calls, such as `table.group_by(...).select(...)`.
+
+The [Table API]({{ site.baseurl }}/dev/table/tableApi.html?code_tab=python) documentation describes all Table API operations that are supported on streaming and batch tables.
```

Comment:
Two link styles are used. Could you unify them and only use **link xxx.md** in all the places.